### PR TITLE
Windows xbmgmt flash scan

### PR DESCRIPTION
**Pull request by AShivangi**
Sample outputs:

```
C:\Users\sagarw\XRT>xbmgmt.exe --override flash --scan
Card [0]:
        Card type:              u250
        Flash type:             SPI
        Flashable partition running on FPGA:
                xilinx_u250_xdma_201830_2,[ts=0x5d027a10],[SC=3.1]
        Flashable partitions installed in system:
                None

C:\Users\sagarw\XRT>xbmgmt.exe --override flash --scan -v
Card [0]:
        Card type:              u250
        Flash type:             SPI
        Flashable partition running on FPGA:
                xilinx_u250_xdma_201830_2,[ts=0x5d027a10],[SC=3.1]
        Flashable partitions installed in system:
                None
        Card name                       AU250A64G
        Card S/N:                       21320493800R
        Config mode:            7
        Fan presence:           A
        Max power level:                225W
        MAC address0:           00:0A:35:05:FE:60
        MAC address1:           00:0A:35:05:FE:61
        MAC address2:           FF:FF:FF:FF:FF:FF
        MAC address3:           FF:FF:FF:FF:FF:FF
```

### CODE REVIEW

**Comment by maxzhen** [comment]
Do you want to change "ts" to "ID" to keep it in sync w/ output on Linux?

File: src/runtime_src/core/pcie/windows/device_windows.cpp [hunk 1]
```diff
@@ -97,14 +97,6 @@ rom(const device_type* device, qr_type qr, const std::type_info&, boost::any& va
   case qr_type::QR_ROM_FPGA_NAME:
     value = std::string(reinterpret_cast<const char*>(hdr.FPGAPartName));
     return;
-  }
```
Comment (stsoe) [inline src/runtime_src/core/pcie/windows/device_windows.cpp]: Remove this check for device type will not work.  On the user side  there is only limited ROM info, namely the first 4 query values.  Alas, if you ask for QR_ROM_UUID and EPOCH on the user side, then an exception should be thrown.

Comment (AShivangi) [reply src/runtime_src/core/pcie/windows/device_windows.cpp]: Oh. I see.. I was getting an error with this code on mgmt side. I'll look more closely

**Comment by AShivangi** [comment]
Max, I'll do that. Thanks!

**Pull request merged**